scripts/test_assign.py

```
@@ -0,0 +1,173 @@
+        mock_graphql_response = MagicMock()
+        mock_graphql_response.status_code = 200
+        mock_graphql_response.json.return_value = {"data": {"repository": {"issue": {"timelineItems": {"nodes": []}}}}}
+        mock_requests.post.return_value = mock_graphql_response
+
+        mock_rest_response = MagicMock()
+        mock_rest_response.json.return_value = {"total_count": 0, "items": []}
+        mock_requests.get.return_value = mock_rest_response
+
+        result, pr_num = has_open_pr_for_issue(self.owner, self.repo, self.issue_number, self.headers)
+
+        self.assertFalse(result)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertFalse`

Replace `assertFalse(...)` with `assert ...`

(PT009)